src/getMovieReviews.js

```diff
@@ -0,0 +1,69 @@
+import fetch from "node-fetch";
+
+//fetch all reviews for a movie when there are more reviews than in the first fetch
+async function xloadMovieReviews(id, number) {
+    const API_BASE = "https://plankton-app-xhkom.ondigitalocean.app/api";
+    const Q_MOVIE = "filters[movie]=";
+    const Q_PSIZE = "pagination[pageSize]=";
```

Comment:
This code is hard-coding API stuff, outside of any CMS adapter. That means that this type of code now exists in more than one place, and if you wanted to use a separate API environment (e.g. move from a temporary development environment to the real environment) you would have to find all of those places and change them.